Dockerfile

```diff
@@ -17,7 +17,7 @@ COPY . ./
 
 RUN CGO_ENABLED=0 go build -a -o build/migrate.linux-386 -ldflags="-s -w -X main.Version=${VERSION}" -tags "$(DATABASE) $(SOURCE)" ./cmd/migrate
 
-FROM alpine:3.16
+FROM alpine:latest
```

Comment:
We should not use latest as we won't be able to definitively know what that version was actually used.

Reply:
I considered that when we started upgrading alpine base images and rebuilding images for security fixes, but it seemed like a pretty minor edge case where the specific base version of alpine would be relevant, so I didn't see any issues with using `latest`. Are there cases you have in mind, or is it just a best practices concern?

Reply:
Say :latest points to 3.18 today. So we kick off a build and our version 2.0.0 of our image has a base image of alpine:latest which is 3.18. Then we find an issue with alpine 3.18, which is inside out 2.0.0 image. What do we do to fix it? What code change do you introduce to reproduce the issue? Triggering a new build won't change the version of the migrate image but will change the version of the base image.